doc/classes/Camera2D.xml

```diff
@@ -371,13 +375,13 @@
 			The camera's offset, useful for looking around or camera shake animations.
 		</member>
 		<member name="rotating" type="bool" setter="set_rotating" getter="is_rotating">
-			If [code]true[/code] the camera can rotate during gameplay. Default value: [code]false[/code].
+			If [code]true[/code] the camera rotates with the target.
 		</member>
 		<member name="smoothing_enabled" type="bool" setter="set_enable_follow_smoothing" getter="is_follow_smoothing_enabled">
-			If [code]true[/code] the camera's velocity changes gradually, preventing sudden starts and stops. Default value: [code]false[/code].
+			If [code]true[/code] the camera smoothly moves towards the target at [member smoothing_speed].
 		</member>
 		<member name="smoothing_speed" type="float" setter="set_follow_smoothing" getter="get_follow_smoothing">
-			How fast the camera's velocity changes if smoothing is enabled.
+			Speed in pixels per second of the smoothing effect of the camera.
```

Comment:
- Use possessive over "of the"

"of the camera's smoothing effect."

- add link to associated member:

See [member smoothing_enabled].